### PR TITLE
test: journeyquicksettingschat

### PR DESCRIPTION
**Pull request by jianwei1**
journeyquicksettings chat tests

### CODE REVIEW

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/JourneyAppearance/Host/HostForm/HostForm.spec.tsx [hunk 1]
```diff
@@ -204,4 +236,216 @@ describe('HostForm', () => {
...
+            id: 'hostId',
+            __typename: 'Host',
+            title: 'Title',
+            location: 'USA',
+            src1: 'imageSrc1',
+            src2: 'imageSrc2'
+          }
+        }
+      }
+    }
```
Comment (edmonday) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/JourneyAppearance/Host/HostForm/HostForm.spec.tsx:264]:
```suggestion
      result: jest.fn(() => {
              data: {
          hostUpdate: {
            id: 'hostId',
            __typename: 'Host',
            title: 'Title',
            location: 'USA',
            src1: 'imageSrc1',
            src2: 'imageSrc2'
          }
        }
      })
```

This way you can get rid of line 266

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/JourneyAppearance/Host/HostForm/HostForm.spec.tsx [hunk 2]
```diff
@@ -204,4 +236,216 @@ describe('HostForm', () => {
...
+
+    const mockHostUpdateResult = jest
+      .fn()
+      .mockReturnValue(updateHostMock.result)
```
Comment (edmonday) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/JourneyAppearance/Host/HostForm/HostForm.spec.tsx:268]:
```suggestion
```

drop

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/JourneyAppearance/Host/HostForm/HostForm.spec.tsx [hunk 3]
```diff
@@ -204,4 +236,216 @@ describe('HostForm', () => {
...
+          />
+        </JourneyProvider>
+      </MockedProvider>
+    )
+
+    const input = screen.getByRole('textbox', { name: 'Host Name' })
+    fireEvent.change(input, { target: { value: 'Someone' } })
+    fireEvent.submit(input)
+    await waitFor(() => {
+      expect(mockHostUpdateResult).toHaveBeenCalled()
```
Comment (edmonday) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/JourneyAppearance/Host/HostForm/HostForm.spec.tsx:289]:
```suggestion
      expect(updateHostMock.result).toHaveBeenCalled()
```

**Review by edmonday — CHANGES_REQUESTED** [review]

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/JourneyAppearance/Host/HostForm/HostForm.spec.tsx (hunk 1 above)
Comment (jianwei1) [reply apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/JourneyAppearance/Host/HostForm/HostForm.spec.tsx:264]: After chatting with Nisal, we're going to keep this as is

**Review by edmonday — APPROVED** [review]

**Pull request merged**